dashboard/public/frequency/frequency.html

```diff
@@ -0,0 +1,61 @@
+<link type="text/css" rel="stylesheet" href="/frequency/frequency.css">
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.5/d3.min.js"></script>
```

Comment:
I wonder if it's better to bundle this in `/dashboard/vendor/assets/javascripts`?  @philbogle may have guidance here.  Traditionally we've tried to server most of our content through just a few domains, since schools can be aggressive about blocking external content.  See: https://code.org/educate/it.